packages/components-js/src/pages/segmented-control.html

```diff
@@ -52,6 +52,54 @@
   </p-segmented-control>
 </div>
 
+<p-segmented-control class="playground light" title="should render segmented-control with columns=1" columns="1">
+  <p-segmented-control-item value="s">S</p-segmented-control-item>
```

Comment:
What about an example with more text that leads to a line break and not just single characters?
Also it might help if there would be the text columns=1 visible in the screenshot to simplify finding the right spot.